### DISCUSSION TITLE
Tailwind stopped working after a while

### DISCUSSION THREAD

**Discussion by theCatelyna** [tailwindlabs/tailwindcss#15331] [Help] [answered]
Hi,
I've been working on a next.js project for about a month now.
Yesterday the project was working without any issues, but when I opened it today no tailwind stylings showed up. They are still visible in the inspector, but not on the website.
I tried updating tailwind, but that only caused problems with other dependencies which I can't resolve.
This happened to me once before so I just started a new project because I thought I messed up some config-files, but this time I'm sure it is not my fault since it happened overnight without me changing anything. I am the only person working on this project so no one else messed with the files.
Uninstalling and reinstalling unfortunately did nothing.
Does anyone have an idea why this happened out of nowhere and how to resolve it?

Caty

**Comment by wongjn** [comment]
Consider checking the `content` file globs sufficiently cover template files that use Tailwind class names. Otherwise, consider providing a remotely-hosted git repository of a project that reproduces the unexpected behavior.

**Reply by theCatelyna** [reply]
I don't know what you mean by template files so here is my repository: https://github.com/theCatelyna/cis-project.git

**Reply by wongjn** [reply]
Seems to be working as expected for me:
![image](https://github.com/user-attachments/assets/e3f279b8-962e-46f0-8256-a5cbcdf09d6e)

**Reply by theCatelyna** [reply]
It seems to be a problem with the path or the components then. "/location/[location]" looks like its working (calendar and stuff) but when you change the date the container I made for the schedule shows up in plain html

**Comment by theCatelyna** [comment, marked as answer]
It works again!
The colors just didn't show up (the rest of tailwind did though).
I don't know how it was fixed (I just played with colors on a test page and it worked for the other pages afterwards?), but I'm glad it works again.
